### ISSUE TITLE
Any plans to add blame command?

### ISSUE THREAD

**Issue by dkns** [jreybert/vimagit#59]
Hi, awesome plugin! Thanks a lot for creating it. One thing I would love to see is blame command implemented. Any plans to add it in the future?

**Comment by fpnick** [comment]
+1

**Comment by jreybert** [comment]
Hi

[fugitive](https://github.com/tpope/vim-fugitive) and [vim-gita](https://github.com/lambdalisue/vim-gita) already implement nice git blame feature (gita's one is really nice, have a look).

Actually, both existing solutions seems OK to me. I don't intend to copy something working well. But if you have any idea of a git blame integrating well in the magit workflow, let me know!

I'll check how emacs magit handles got blame, to see if it is inspiring.

**Comment by dkns** [comment]
Yeah, vim-gita is awesome but I'd rather not load separate vim plugin just to have nice looking blame interface. Let me know if you need any help with developing this one.

**Comment by jreybert** [comment]
As I don't see what vimagit could bring more for git blame vim integration, I don't plan to implement it.

> This is the Unix philosophy: Write programs that do one thing and do it well.

If you can see a way where the vimagit workflow could integrate a git blame output in an interesting manner, I would be happy to see it! Feel free to reopen this ticket in this case.

**Comment by kristijanhusak** [comment]
@jreybert why vimagit has to bring more to git blame then fugitive or gita? Most of people want to use only one git plugin, and as @dkns noted, pulling in another plugin just to use single feature is redundant.
It's same for me, only thing i use from fugitive is blame, since i don't like the other parts, but here i like everything, but only missing this blame feature. I agree with Unix philosophy, but i don't think it applies here. It would apply if we would have a plugin that is used only for blame, like a vim-fugitive-blame, but that's not the case, fugitive has a lot of it's own features.

**Comment by jreybert** [comment]
Hi @kristijanhusak .

You are quite right, use only one plugin would be better. But:
-  I don't have the time to add features which do not exist in other plugins (#23, #142, #83, one day handle rebase like magit), mimic existing behavior is a very low priority
- fugitive is not that huge, loading it at vim startup is not noticeable. Unify fugitive blame mapping with vimagit should be easy.

So, I am sorry, but integrate a blame feature in vimagit is very low priority...

**Comment by kristijanhusak** [comment]
@jreybert I understand that it's low priority, and that's completely fine, but rejecting it should not be a case. @dkns offered to contribute, so if someone else can do it, why not.

**Comment by jreybert** [comment]
You make a point. I remove the rejected flag and reopen the issue.

@dkns, if you are still interested to develop this new feature, feel free!
